infrastructure/stacks/capacity_management/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "project" {
+  description = "Project code typically reflects sub project of project owner eg nhse-uec-dos"
+}
+variable "project_owner" {
+  description = "Project owner based on orgnaistion and department code eg nhse-uec"
+}
+variable "environment" {
+  description = "The environment - dev, test, staging etc"
+}
+variable "organisations_table_name" {
+  default = "organisations"
+}
+variable "locations_table_name" {
+  default = "locations"
+}
+variable "healthcare_services_table_name" {
+  default = "healthcare_services"
```

Comment:
One for wider discussion ... in the past we have avoided setting defaults in the variables.tf preferring to have them set in some sort of common.tfvars file ; we then overwrite them in env specific tfvars files. I think the rationale was that we can see our default values in one place rather than looking for them in various variables.tf files ... not sure which way to go ... happy to discuss